docs/topics/auth/default.txt

```diff
@@ -1665,6 +1665,14 @@ provides several built-in forms located in :mod:`django.contrib.auth.forms`:
     sets the user's password using
     :meth:`~django.contrib.auth.models.User.set_password()`.
 
+.. class:: UserCreationForm
+
+    Inherits from BaseUserCreationForm. Adds an additional method:
+
+    :meth:`~django.contrib.auth.forms.UserCreationForm.clean_username()`:
```

Comment:
You cannot add a reference to sth that is not documented.

Reply:
Makes sense. Will rebase and push again.